modules/apps/journal/journal-service/src/main/java/com/liferay/journal/util/impl/JournalContentImpl.java

```diff
@@ -224,7 +224,7 @@ public JournalArticleDisplay getDisplay(
 
 			secure = themeDisplay.isSecure();
 
-			if (!lifecycleRender) {
+			if (!lifecycleRender && (portletRequestModel != null)) {
```

Comment:
@jcampoy Your commit message is "Be conservative, only have into account the themeDisplay lifecycle when there is not portletModelRequest", it sounds like you want to do "portletRequestModel == null" here, right? And make sure you squash your commits, to keep the history clear.

Sorry, I don't know how to answer your question, I barely know any logic here, you will need to go ask journal guys. I am just here to ensure your change is not self conflicting.

Reply:
Yes! sorry by the mistake! You are right It is "portletRequestModel == null", I will resend the pull request with squash.

I will try to talk with the journal guys and manage that in other ticked if it is needed.

Thanks!